packages/elements/src/components/ui/cardy/Cardy.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, BoxProps } from "@stenajs-webui/core";
+import cx from "classnames";
+import * as React from "react";
+import { PropsWithChildren } from "react";
+import styles from "./Cardy.module.css";
+import { cssColor, CssPropColor } from "@stenajs-webui/theme";
+import { Spinner } from "../spinner/Spinner";
+
+interface CardyProps extends Pick<BoxProps, "minWidth"> {
+  color?: CssPropColor;
+  loading?: boolean;
+  noPaddingBottom?: boolean;
+  className?: string;
+}
+
+export const Cardy: React.FC<PropsWithChildren<CardyProps>> = ({
+  children,
+  color = "--lhds-color-ui-200",
+  loading,
+  noPaddingBottom,
+  className,
+  ...boxProps
+}) => {
+  return (
+    <div style={{ position: "relative" }}>
```

Comment:
A random relative div block element seems to be asking for problems.
Why position relative and why not a box or fragment?